tests/mu_define_enum_test.c

```diff
@@ -34,5 +42,13 @@ int run_mu_define_enum_tests(void)
 
     POOR_MANS_ASSERT(MU_ENUM_VALUE_COUNT(TEST4_ENUM_VALUES) == 3);
 
+    const char* TEST5_test5_b = MU_ENUM_TO_STRING(TEST5_ENUM, 42);
+    POOR_MANS_ASSERT(TEST5_test5_b != NULL);
+    POOR_MANS_ASSERT(strcmp("test5_b = 42", TEST5_test5_b) == 0);
```

Comment:
>test5_b = 42 [](start = 29, length = 12)

would have expected that only "test5_b" is printed.

Reply:
I guess with enough developer effort that would also be possible. I actually liked the fact that the values are there too ... I have to say that ideally would have liked something like test5_b (42). That's maybe for another change.

---
In reply to: [328351122](https://github.com/Azure/azure-macro-utils-c/pull/30#discussion_r328351122) [](ancestors = 328351122)